packages/ra-ui-materialui/src/layout/Notification.tsx

```diff
@@ -22,6 +22,7 @@ export const Notification = (
         classes: classesOverride,
         type = 'info',
         autoHideDuration = 4000,
+        anchorOrigin = { vertical: 'bottom', horizontal: 'center' },
```

Comment:
You should create a constant outside of the component wit hthat value, otherwise the value will be recreated at each render and cause unnecessary renders of the underlying Snackbar component.

Reply:
@fzaninotto  I'm not sure I understood your proposal, but do you expect me to remove this property from props?


```js
    const {
        classes: classesOverride,
        type = 'info',
        autoHideDuration = 4000,
        className,
        ...rest
    } = props;
...
    const anchorOrigin =  { vertical: 'bottom', horizontal: 'center' }
```

or

```js
    const {
        classes: classesOverride,
        type = 'info',
        autoHideDuration = 4000,
        className,
        anchorOrigin
        ...rest
    } = props;
...
    const { vertical, horizontal } =  anchorOrigin
```

Well, it was validating it can also be done this way 

```js
Notification.defaultProps = {
    anchorOrigin: { vertical: 'bottom', horizontal: 'center' },
};
```

this way I could even remove the prop anchorOrigin in line 90, because from what I was looking has `{...rest}`



> Please document the additional prop in the commponent's JSDoc

Where is this docs? Theming.md? If not, where should I create?

Reply:
No, he meant to declare a const outside the component:

```
const defaultAnchorOrigin = { vertical: 'bottom', horizontal: 'center' }
```

and to use that as the default for the prop:
```
anchorOrigin = defaultAnchorOrigin
```

Reply:
Done @djhi @fzaninotto  https://github.com/marmelab/react-admin/pull/6878/commits/cfaaa01312e91590f4879b48cd6a92c2a5f1077b

Where is JSDoc?